Form-Controls/index.html

```diff
@@ -3,72 +3,85 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="description" content="This is a form of ordering T-Shirt" />
+    <meta name="description" content="This is a form for ordering T-Shirts" />
     <link rel="stylesheet" href="styles.css" />
     <title>T-Shirt Order Form</title>
   </head>
   <body>
-    <div class="form-container">
+    <header>
       <h1 class="form-title">Order Your T-Shirt</h1>
-      <form class="order-form" id="orderForm">
-        <div class="form-group">
-          <label class="form-label" for="name">Name</label>
-          <input
-            type="text"
-            class="form-input"
-            id="name"
-            name="name"
-            required
-          />
-        </div>
-        <div class="form-group">
-          <label class="form-label" for="email">Email</label>
-          <input
-            type="email"
-            class="form-input"
-            id="email"
-            name="email"
-            required
-          />
-        </div>
-        <div class="form-group">
-          <label class="form-label" for="color">Choose the T-Shirt color</label>
+    </header>
 
-          <select class="form-select" id="color" name="color" required>
-            <option value="" disabled selected>Select a color</option>
-            <option value="red">Red</option>
-            <option value="blue">Blue</option>
-            <option value="green">Green</option>
-          </select>
-        </div>
-        <div class="form-group">
-          <label class="form-label" for="size">Choose the T-Shirt size</label>
-          <select class="form-select" id="size" name="size" required>
-            <option value="" disabled selected>Select a size</option>
-            <option value="XS">XS</option>
-            <option value="S">S</option>
-            <option value="M">M</option>
-            <option value="L">L</option>
-            <option value="XL">XL</option>
-            <option value="XXL">XXL</option>
-          </select>
-        </div>
-        <div class="form-group">
-          <label class="form-label" for="delivery-date"
-            >When do you want it delivered?</label
-          >
-          <input
-            type="date"
-            class="form-input"
-            id="delivery-date"
-            name="date"
-            min="2024-06-05"
-            max="2030-07-03"
-            required
-          />
-        </div>
-        <button type="submit" class="form-button">Submit Order</button>
-      </form>
-    </div>
+    <main>
+      <div class="form-container">
+        <form class="order-form" id="orderForm">
+          <div class="form-group">
+            <label class="form-label" for="name">Name</label>
+            <input
+              type="text"
+              class="form-input"
+              id="name"
+              name="name"
+              required
+              minlength="2"
+              pattern="[A-Za-z\s]+"
```

Comment:
Your pattern allows only letters and spaces, which is good for names. However, consider if you want to allow hyphens or apostrophes (e.g., for names like "O'Connor" or "Anne-Marie").
You can test your pattern here - https://regexr.com

<img width="565" alt="Screenshot 2024-10-18 at 00 15 56" src="https://github.com/user-attachments/assets/62b4d1a4-a63e-497b-97b6-d4fb6a8ff180">